alerts/add-runbook-links.libsonnet

```diff
@@ -16,7 +16,7 @@ local lower(x) =
   prometheusAlerts+::
     local addRunbookURL(rule) = rule {
       [if 'alert' in rule then 'annotations']+: {
-        runbook_url: $._config.runbookURLPattern % lower(rule.alert),
+        [if !('runbook_url' in rule.annotations) then 'runbook_url']: $._config.runbookURLPattern % lower(rule.alert),
```

Comment:
Actually this is really nice, but right now we just want to remove *all* runbook links because no actual runbooks have been written, but we still link to an empty page.